README.md

```diff
@@ -18,3 +18,8 @@ See the [contributing guide](CONTRIBUTING.md) for information on what you can do
 
 ### Data forensics
 * Ability to parse `Registry.pol` files from Windows machines
+
+### Math / machine learning / data scienc
+* Sparse matrix libraries (SPRS library needs a lot of love)
```

Comment:
I think these are excellent starts, from what I know of the linear algebra and machine learning ecosystem in Rust. I'm a little hesitant to merge because I worry that what's listed here might not be actionable enough for library writers (which which you may disagree -- I'm unfamiliar with these domains). For instance:

> SPRS [assuming [here](https://github.com/vbarrielle/sprs)) needs a lot of love

What's missing? Have issues already been submitted for the `love` that the library needs and/or do you think the library has an incompatible design for what you had in mind? If issues already exist, it would be great if we could link those too -- that'd get more attention on them! 

> rust-learn and rusty-machine are insufficient

Similar thoughts with finding specific use cases -- what is it that's NOT there right now? How could we make those libraries "sufficient"? Are there any issues that have gone unresolved?

As another general thought, if you have a lot of details in mind but don't necessarily want to clutter this list, I wouldn't mind keeping the elaborate details in an issue on this repo so somebody else can find them. :)